### ISSUE TITLE
`load_checkpoint` is not implemented in SequenceTagger?

### ISSUE THREAD

**Issue by himkt** [zalandoresearch/flair#1209]
**Describe the bug**
When I tried to resume training by following [a documentation](https://github.com/zalandoresearch/flair/blob/64dabf719191a2a15c2ebacfb8d6965cd9289230/resources/docs/TUTORIAL_7_TRAINING_A_MODEL.md#resuming-training), I got an error like below.

```
Traceback (most recent call last):
  File "demo.py", line 49, in <module>
    checkpoint = tagger.load_checkpoint(Path('resources/taggers/example-ner/checkpoint.pt'))
  File "/home/ubuntu/.local/share/virtualenvs/dlhacks1021-ZzrrVfN-/lib/python3.7/site-packages/torch/nn/modules/module.py", line 585, in __getattr__
    type(self).__name__, name))
AttributeError: 'SequenceTagger' object has no attribute 'load_checkpoint'
```

**To Reproduce**

Exactly the same as documentation.

<details>

```python
from flair.data import Corpus
from flair.datasets import WNUT_17
from flair.embeddings import TokenEmbeddings, WordEmbeddings, StackedEmbeddings
from typing import List

# 1. get the corpus
corpus: Corpus = WNUT_17().downsample(0.1)

# 2. what tag do we want to predict?
tag_type = 'ner'

# 3. make the tag dictionary from the corpus
tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)

# 4. initialize embeddings
embedding_types: List[TokenEmbeddings] = [
    WordEmbeddings('glove')
]

embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)

# 5. initialize sequence tagger
from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                        embeddings=embeddings,
                                        tag_dictionary=tag_dictionary,
                                        tag_type=tag_type,
                                        use_crf=True)

# 6. initialize trainer
from flair.trainers import ModelTrainer
from flair.training_utils import EvaluationMetric

trainer: ModelTrainer = ModelTrainer(tagger, corpus)

# 7. start training
trainer.train('resources/taggers/example-ner',
              learning_rate=0.1,
              mini_batch_size=32,
              max_epochs=1,
              checkpoint=True)

# 8. stop training at any point

# 9. continue trainer at later point
from pathlib import Path

checkpoint = tagger.load_checkpoint(Path('resources/taggers/example-ner/checkpoint.pt'))
trainer = ModelTrainer.load_from_checkpoint(checkpoint, corpus)
trainer.train('resources/taggers/example-ner',
              learning_rate=0.1,
              mini_batch_size=32,
              max_epochs=1,
              checkpoint=True)
```

</details>

**Expected behavior**
It loads snapshot and resumes training.

**Screenshots**
<img width="1787" alt="Screen Shot 2019-10-14 at 15 13 28" src="https://user-images.githubusercontent.com/5164000/66732024-32b87d00-ee95-11e9-97d9-6a7e1b4f12f4.png">

**Environment (please complete the following information):**
 - OS [e.g. iOS, Linux]: Ubuntu 18.04.3 LTS
 - Version [e.g. flair-0.3.2]: master branch e8c0afe

**Additional context**

I can see `load_checkpoint` in `LanguageModel` but can't in `SequenceTagger`. 🤔 
https://github.com/zalandoresearch/flair/blob/799c8dc93608280b11f358b7923eda989bcc72a7/flair/models/language_model.py#L204-L234

**Comment by alanakbik** [comment]
Hello @himkt - we changed the syntax for loading checkpoints for the `Model` class. You can now load checkpoints only through the `ModelTrainer` with one line of code, see #1101: 

```python
# 3. load last checkpoint with one line of code
trainer = ModelTrainer.load_checkpoint('experiment/folder/checkpoint.pt', corpus)
```

**Comment by himkt** [comment]
@alanakbik Thanks, it makes sense and I created PR to fix the tutorial #1211.
